doc/upgrading.md

```diff
@@ -134,6 +137,9 @@ Replace any joda class usage by a java.time class usage. RMB no longer ships wit
 joda-time that is deprecated because the replacement java.time has been in JDK core
 since Java 8.
 
+Add commons-lang:commons-lang:2.6 dependency to pom.xml if commons-lang is used.
```

Comment:
Has to be that `commons-lang:2.6` specific version?

Reply:
Yes, that's the latest 2.* version: https://mvnrepository.com/artifact/commons-lang/commons-lang

Reply:
> Yes, that's the latest 2.* version: https://mvnrepository.com/artifact/commons-lang/commons-lang

Would https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 not work?

Reply:
https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/index.html resides in org.apache.commons.lang package.
https://commons.apache.org/proper/commons-lang/javadocs/api-release/index.html resides in org.apache.commons.lang3 package.